contracts/bridge/BridgeOperator.sol

```diff
@@ -23,6 +23,8 @@ contract BridgeOperator is Ownable {
         for (uint8 i = 0; i < uint8(VoteType.Max); i++) {
             operatorThresholds[uint8(i)] = 1;
         }
+
+        operators[msg.sender] = true;
```

Comment:
It seems only for testing purposes. In case of using `subbridge.deploy` API, it would be good to support the registerOperator API. I'll need a revert this PR later.

Reply:
Right. Let's think better UX in next time.